.gitignore

```diff
@@ -1,5 +1,6 @@
 **/*.pyc
 **/.pytest_cache
+.vscode/
```

Comment:
btw, protip: if you make a `.gitignore` in your `~/` then it will be applied to all your repos. It's a good place to put IDE-related stuff.

Reply:
Wow, awesome tip! thx